### PR TITLE
DOCSP-38514-Create-Javascript-Language-Landing-Page

### PR DESCRIPTION
**Pull request by mmaville-mdb**
Setting up structure for project and adding the Python landing page.

**For Reviewer/SME**:

- Links for Prisma and MongooseLi were added by me and need to be verified by an SME to ensure the intended target links.

Staging link: https://preview-mongodbmmavillemdb.gatsbyjs.io/landing/DOCSP-38514-Create-Javascript-Language-Landing-Page/languages/javascript/

Build log: https://workerpool-boxgs.mongodbstitch.com/pages/job.html?collName=queue&jobId=662ad493b67dbf804a56a331

### CODE REVIEW

File: source/languages/javascript.txt [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+=======================
+MongoDB with JavaScript
+=======================
+
+.. default-domain:: mongodb
+
+.. introduction::
+    
+   Develop lightweight Javascript or TypeScript applications backed by a 
+   `MongoDB Atlas cloud database <https://www.mongodb.com/docs/atlas/>`__.
```
Comment (mongoKart) [inline source/languages/javascript.txt:14]:
```suggestion
   Develop lightweight Javascript or TypeScript applications backed by 
   MongoDB's `Atlas cloud database <https://www.mongodb.com/docs/atlas/>`__. 
```

File: source/languages/javascript.txt [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+
+.. introduction::
+    
+   Develop lightweight Javascript or TypeScript applications backed by a 
+   `MongoDB Atlas cloud database <https://www.mongodb.com/docs/atlas/>`__. 
+   Developer integrations like Prisma and Mongoose support your MERN and 
+   MEAN stack applications. MongoDB's Node.js driver is written in 
+   TypeScript and offers all the quality of life improvements modern 
+   developers expect such as predictability, better readability, rich IDE 
+   support and strong type safety.
```
Comment (mongoKart) [inline source/languages/javascript.txt:19]:
```suggestion
   TypeScript and offers all the quality-of-life improvements modern 
   developers expect, such as predictability, better readability, rich IDE 
   support, and strong type safety.
```

File: source/languages/javascript.txt [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+      :icon-alt: Driver card icon
+
+      Provides a TypeScript API for working with MongoDB databases consistent with other MongoDB language drivers.
+
+Integrations
+------------
+
+`Prisma → <https://www.mongodb.com/developer/languages/typescript/type-safety-with-prisma-and-mongodb/>`__
+
+`MongooseLi → <https://www.mongodb.com/developer/languages/javascript/getting-started-with-mongodb-and-mongoose/>`__
```
Comment (mongoKart) [inline source/languages/javascript.txt:54]: Typo I think?

```suggestion
`Mongoose → <https://www.mongodb.com/developer/languages/javascript/getting-started-with-mongodb-and-mongoose/>`__
```

File: source/languages/javascript.txt [hunk 4]
```diff
@@ -0,0 +1,85 @@
...
+
+More ways to learn
```
Comment (mongoKart) [inline source/languages/javascript.txt:56]:
```suggestion
More Ways to Learn
```

**Review by mongoKart — COMMENTED** [review]
few things

File: source/languages/javascript.txt [hunk 5]
```diff
@@ -0,0 +1,85 @@
...
+   :columns: 3
+   :style: large-icon
+
+   .. card::
+      :headline: View Javascript developer resources
+      :url: https://www.mongodb.com/developer/languages/javascript/
+      :icon: /images/technical-mdb-devhub.svg
+      :icon-alt: DevHub icon
+
+      Python Resources  →
```
Comment (caitlindavey) [inline source/languages/javascript.txt:69]:
```suggestion
      Javascript Resources  →
```

File: source/languages/javascript.txt [hunk 6]
```diff
@@ -0,0 +1,85 @@
...
+
+   .. card::
+      :headline: Take the Getting Started with Javascript course 
+      :url: https://learn.mongodb.com/learning-paths/mongodb-nodejs-developer-path
+      :icon: /images/icons/university.svg
+      :icon-alt: Learn icon
+
+      Python Developer Path →
```
Comment (caitlindavey) [inline source/languages/javascript.txt:77]:
```suggestion
      Javascript Developer Path →
```

**Pull request merged**